src/bin/pgcopydb/ld_stream.c

```diff
@@ -313,6 +313,25 @@ stream_init_context(StreamContext *privateContext, StreamSpecs *specs)
 
 	privateContext->lastWrite = 0;
```

Comment:
I know it's not directly in your PR, but please consider renaming `privateContext->lastWrite` to `privateContext->lastWriteTime` or something, so that we avoid confusion with `maxWrittenLSN`.

Reply:
Done